tests/swift-sdk-swift-tests/in-app-tests/InAppPersistenceTests.swift

```diff
@@ -57,4 +57,70 @@ class InAppPersistenceTests: XCTestCase {
             XCTFail("conversion to string failed")
         }
     }
+    
+    func testPersistentReadStateAfterLogout() {
```

Comment:
This test is actually not doing what it is supposed to. We need to make sure that server state is reflected on logout and re-login. Here the server state is `read = false`. 

The test is passing for wrong reasons. Please remove this test.